communities.json

```diff
@@ -771,6 +771,13 @@
       "quote": "Discord has been amazing for accelerating the development of React Spring and React Three Fiber. Screen sharing and instant communication have been pivotal in resolving bugs and coordinating with our contributors.",
       "inviteCode": "poimandres",
       "githubUrl": "https://github.com/pmndrs"
+    },
+    {
+      "logo": "magma.png",
```

Comment:
This logo is just going to look like a square with rounded corners when the [website filter](https://discord.com/open-source) is applied, do you have a version of the logo that makes use of transparency to distinguish the various shades of gray?

Reply:
I don't